pkg/protocompat/message_test.go

```diff
@@ -104,7 +105,15 @@ func TestMarshalTextString(t *testing.T) {
 ` + `cluster_id: "aaaaaaaa-bbbb-4011-0000-111111111111"
 ` + `cluster_name: "Cluster 1"
 `
-	assert.Equal(t, expectedString, asString)
+	// String output is not guarantied.
+	// Info: https://pkg.go.dev/google.golang.org/protobuf@v1.34.1/encoding/prototext#Format
+	// There is randomization added to output to ensure that library users
+	// are not relaying on stable output format.
+	// Info: https://go-review.googlesource.com/c/protobuf/+/151340
+	re := regexp.MustCompile(`: +"`)
```

Comment:
Is it working with non string types?

Reply:
I have changed this as @rhybrillou suggested. To use "expected regex" instead of string.